ofctrl/fgraphFlow.go

```diff
@@ -471,7 +496,9 @@ func (self *Flow) install() error {
 	self.Table.Switch.Send(flowMod)
 
 	// Mark it as installed
+	self.lock.Lock()
 	self.isInstalled = true
+	self.lock.Unlock()
```

Comment:
Today, we don't have many cases where we modify a flow after its installed. This could change in future. I wanted to read/write protect the flow in case we modify it later. May be i should take flow level locks for all public APIs instead of doing it in inner functions..